docs: how to add a missing dependency to langflow desktop (#8942)
langflow-ai/langflow: docs/docs/Develop/install-custom-dependencies.md

uv pip install "langflow[deploy,local,postgresql]"
```

## Install custom dependencies in Langflow Desktop {#langflow-desktop}

> To add dependencies to **Langflow Desktop**, add an entry for the package to the application's `requirements.txt` file.

Reviewer: ```suggestion
To add dependencies to Langflow Desktop, add an entry for the package to the application's `requirements.txt` file:
```

    * On macOS, the file is located at `/Users/USER/.langflow/data/requirements.txt`.
    * On Windows, the file is located at `C:\Users\USER\AppData\Roaming\com.Langflow\data\requirements.txt`.

> Add the dependency and version to `requirements.txt` in the following format:
> 
>     ```
>     docling==2.40.0
>     ```

Reviewer: ```suggestion
Add the dependency and version to `requirements.txt` on separate lines in the format `PACKAGE==VERSION`, such as `docling==2.40.0`.
```

> Restart Langflow desktop to install the dependency.

Reviewer: ```suggestion
Restart Langflow Desktop to install the dependencies.
```

> 

Reviewer: ```suggestion

If you need to change or uninstall custom dependencies. edit the `requirements.txt` file, and then restart Langflow Desktop.

```
